CMake/ReleaseDebugAutoFlags.cmake

```diff
@@ -0,0 +1,45 @@
+#
```

Comment:
I'm wondering how relevant is this file.
For example, what is the purpose of FAST? How to enable it? I know of the standard CMake build types (https://cmake.org/cmake/help/v3.0/variable/CMAKE_BUILD_TYPE.html): DEBUG, RELEASE, RELWITHDEBINFO, MINSIZEREL.

What is not ok in this build types that you need to tweak them?

If someone need full/advanced control of the build settings, they will want to roll their own, maybe using a CMake toolchain file?

Reply:
Fine with that. We do not need this mode in abseil, this comes from an other project.

Reply:
Fixed